ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -201,9 +200,9 @@ private enum UnityLayers {
   [KSPField(isPersistant = true)]
   internal MainWindow main_window_;
 
-  public event Action clear_locks;
-  public event Action dispose_windows;
-  public event Action render_windows;
+  public event Action ClearLocks;
+  public event Action DisposeWindows;
+  public event Action RenderWindows;
```

Comment:
Do we really want to change style for events? We had consistendly been using snake_case.

Reply:
Let us go with BactrianCamelCase but noun phrases.

Reply:
Done, but I am unconvinced.